pkg/apis/resource/v1alpha1/cluster/cluster_resource.go

```diff
@@ -123,16 +137,18 @@ func (s *Resource) GetURL() string {
 // Replacements is used for template replacement on a ClusterResource inside of a Taskrun.
 func (s *Resource) Replacements() map[string]string {
 	return map[string]string{
-		"name":      s.Name,
-		"type":      s.Type,
-		"url":       s.URL,
-		"revision":  s.Revision,
-		"username":  s.Username,
-		"password":  s.Password,
-		"namespace": s.Namespace,
-		"token":     s.Token,
-		"insecure":  strconv.FormatBool(s.Insecure),
-		"cadata":    string(s.CAData),
+		"name":                  s.Name,
+		"type":                  s.Type,
+		"url":                   s.URL,
+		"revision":              s.Revision,
+		"username":              s.Username,
+		"password":              s.Password,
+		"namespace":             s.Namespace,
+		"token":                 s.Token,
+		"insecure":              strconv.FormatBool(s.Insecure),
+		"cadata":                string(s.CAData),
+		"clientKeyData":         string(s.ClientKeyData),
```

Comment:
Could you add these to `docs/variables.md` as well?  https://github.com/tektoncd/pipeline/blob/master/docs/variables.md#cluster-pipelineresource

Reply:
I already did that. https://github.com/tektoncd/pipeline/blob/bd5b2fbc5cc915024cd5ba86de5ffb1a729951b9/docs/variables.md